packages/rocketchat-livechat/server/methods/getAgentOverviewData.js

```diff
@@ -0,0 +1,31 @@
+import moment from 'moment';
+
+Meteor.methods({
+	'livechat:getAgentOverviewData'(options) {
+		if (!Meteor.userId() || !RocketChat.authz.hasPermission(Meteor.userId(), 'view-livechat-manager')) {
+			throw new Meteor.Error('error-not-allowed', 'Not allowed', {
+				method: 'livechat:getAgentOverviewData'
+			});
+		}
+
+		if (!(options.chartOptions && options.chartOptions.name)) {
+			console.log('Incorrect analytics options');
+			return;
+		}
+
+		const from = moment(new Date(options.daterange.from));
```

Comment:
This is not a good approach accessing `options.daterange.from`  and `options.daterange.to`  on server side.
If you pass the these data by params will be better.